json/Side_Text.txt

```diff
@@ -9164,1039 +9164,1039 @@
-		"tr_text": "",
+		"tr_text": "Oh, s-sorry...<%br>I was so nervous, I forgot to introduce myself...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "18",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "大丈夫ですよ！<%br>少しずつ、仲良くなりましょう！",
-		"tr_text": "",
+		"tr_text": "That's not a problem!<%br>We'll just get to know each other slowly!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "19",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "そーかもしんねーけど……<%br>こんな調子で、大丈夫かな？",
-		"tr_text": "",
+		"tr_text": "OK, but...<%br>Are you sure this isn't gonna be a problem?",
 		"fileID": 11
 	},
 	{
 		"eventNo": "0",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "任務完了、お疲れ様でした！",
-		"tr_text": "",
+		"tr_text": "Mission complete!<%br>Thank you for your hard work, everyone!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "は、はい……<%br>お疲れ様でした……",
-		"tr_text": "",
+		"tr_text": "O-okay...<%br>Thanks...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "2",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "オマエ、なかなかいい腕してるじゃんかっ！<%br>見直したぜ！",
-		"tr_text": "",
+		"tr_text": "Hey, you're pretty good!<%br>I might have to rethink my opinion of ya!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "3",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "は、はい……<%br>すみません……",
-		"tr_text": "",
+		"tr_text": "O-okay...<%br>Sorry...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "ほめてるのに、なんであやまるんだよ？",
-		"tr_text": "",
+		"tr_text": "What are ya apologizing for?",
 		"fileID": 11
 	},
 	{
 		"eventNo": "5",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "すみません……<%br>その……私、自分に自信が持てなくて……",
-		"tr_text": "",
+		"tr_text": "I'm sorry... for... not being confident...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "6",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "ご迷惑をおかけしないかと<%br>いつも不安なんです……",
-		"tr_text": "",
+		"tr_text": "I keep worrying that I'm bothering you...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "7",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "見事な戦いぶりでしたよ。<%br>そんな不安になる必要ないのに……",
-		"tr_text": "",
+		"tr_text": "You fought very well, considering it was your first<%br>mission. There's really no need to be nervous...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "8",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "そうだよ！<%br>もっと自分に、自信もっていいんだぞ！",
-		"tr_text": "",
+		"tr_text": "Yeah!<%br>You should be more confident in yourself!",
 		"fileID": 11
 	},
 	{
 		"eventNo": "9",
 		"jp_name": "レッドスコルピオ",
 		"tr_name": "Red Scorpio",
 		"jp_text": "は、はい……<%br>すみません……",
-		"tr_text": "",
+		"tr_text": "O-okay...<%br>Sorry...",
 		"fileID": 11
 	},
 	{
 		"eventNo": "10",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "だから……<%br>あやまらなくて、いいんだってば。",
-		"tr_text": "",
+		"tr_text": "For the love of...<%br>Stop apologizing, you're fine.",
```

Comment:
The usage of `da kara` in this case is more of a `I'm telling you...` or `Listen/Look...` instead of a more serious tone.